src/components/SVG/Plus.js

```diff
@@ -8,6 +8,7 @@ const PlusIcon = ({ options }) => {
       fill="currentColor"
       className="bi bi-plus-circle-fill main-icons"
       viewBox="0 0 16 16"
+      onClick={() => setDateFilter({ filter: true })}
```

Comment:
and why is it necessary to broadcast a click on svg, and not on div, which is higher?